lymph/unilateral.py

```diff
@@ -286,6 +296,14 @@ def diag_time_dists(self, new_dists: Union[dict, MarginalizorDict]):
                 f"Cannot use type {type(new_dists)} for marginalization over "
                 "diagnose times."
             )
+        
+    def microscopic_parameter_setter(self, microscopic_parameter):

```

Comment:
In general, use `@property` for something like that, unless setting it is "expensive" in some way or another (see #40 for that).

Reply:
Also this has been adapted already. But I am having problems with the transition matrix right now, since the edges are not being updated whenever a node changes its state. So the newer design will still need some time. (this push was only made since I was working from home on a different computer and needed to push such that I can work at the office)